.circleci/config.yml

```diff
@@ -63,7 +63,9 @@ commands:
           command: |
             VERSION_TO_DEPLOY=$(cat /tmp/build-info/version-to-deploy.txt)
             sed -i "s/appVersion:.*/appVersion: \"${VERSION_TO_DEPLOY}\"/g" "./helm_deploy/prisoner-content-hub-frontend/Chart.yaml"
-            helm upgrade << parameters.releaseName >>-<< parameters.qualifier >> ./helm_deploy/prisoner-content-hub-frontend \
+            QUALIFIER=<< parameters.qualifier >>
+            RELEASE_NAME=<< parameters.releaseName >>${QUALIFIER:+-$QUALIFIER}
+            helm upgrade $RELEASE_NAME ./helm_deploy/prisoner-content-hub-frontend \
```

Comment:
weird syntax is bash string function:: 
https://stackoverflow.com/questions/40771781/how-to-append-a-string-in-bash-only-when-the-variable-is-defined-and-not-null